PLUB/Sources/Views/Home/MeetingSchedule/MeetingScheduleViewController.swift

```diff
@@ -77,32 +78,42 @@ final class MeetingScheduleViewController: BaseViewController {
       .rx.tap
       .asDriver()
       .drive(with: self) { owner, _ in
-        let vc = CreateScheduleViewController(plubbingID: owner.viewModel.plubbingID)
+        let vc = CreateScheduleViewController(viewModel: CreateScheduleViewModel(plubbingID: owner.viewModel.plubbingID))
+        vc.delegate = owner
         owner.navigationController?.pushViewController(vc, animated: true)
       }
       .disposed(by: disposeBag)
-    
+
     let datasource = viewModel.dataSource()
     viewModel.scheduleList
       .drive(tableView.rx.items(dataSource: datasource))
       .disposed(by: disposeBag)
-    
+
     tableView.rx.modelSelected(ScheduleTableViewCellModel.self)
       .withUnretained(self)
       .subscribe(onNext: { owner, data in
         let vc = ScheduleParticipantViewController(
           plubbingID: owner.viewModel.plubbingID,
           data: data
         )
-        vc.delegate = self
+        vc.delegate = owner
         vc.modalPresentationStyle = .overFullScreen
         owner.present(vc, animated: false)
       })
       .disposed(by: disposeBag)
-    
+
     tableView
       .rx.setDelegate(self)
       .disposed(by: disposeBag)
+
+    tableView
+      .rx.contentOffset
+      .compactMap { [weak self] offset in
+        guard let self else { return nil }
+        return (self.tableView.contentSize.height, offset.y)
+      }
```

Comment:
compactMap 내부에 tableView만을 사용하고 있으므로 `[weak self]` 대신 `[tableView]`를 사용하는 것도 좋은 방안이 될 것 같아요!

```suggestion
      .compactMap { [tableView] offset in
        return (tableView.contentSize.height, offset.y)
      }
```

Reply:
이런 방법이 있었다니..바로 수정했습니다~!!